shared/templates/static/bash/mount_option_noexec_removable_partitions.sh

```diff
@@ -0,0 +1,13 @@
+# platform = Red Hat Enterprise Linux 7, multi_platform_fedora
+
+# Include source function library.
+. /usr/share/scap-security-guide/remediation_functions
+
+populate var_removable_partition
+
+NEW_OPT="noexec"
+
+if [ $(grep "$var_removable_partition" /etc/fstab | grep -c "$NEW_OPT" ) -eq 0 ]; then
```

Comment:
If the remediation is being ran, we already know the mount option isn't there. Is the ``if`` check needed?